Source/Core/Core/Debugger/PPCDebugInterface.h

```diff
@@ -19,9 +22,22 @@ class System;
 }  // namespace Core
 class PPCSymbolDB;
 
-void ApplyMemoryPatch(const Core::CPUThreadGuard&, Common::Debug::MemoryPatch& patch,
+void ApplyMemoryPatch(const Core::CPUThreadGuard& guard, std::span<u8> value, const u32 address,
                       bool store_existing_value = true);
 
+template <std::unsigned_integral T>
+void ApplyMemoryPatch(const Core::CPUThreadGuard& guard, T value, const u32 address,
+                      bool store_existing_value = true)
+{
+  u8 data[sizeof(T)];
+  for (u8& b : data | std::views::reverse)
+  {
+    b = static_cast<u8>(value);
+    value >>= 8;
+  }
+  ApplyMemoryPatch(guard, data, address, store_existing_value);
```

Comment:
I don't think `store_existing_value` being `true` would really do anything useful here, it's just going to store the existing value in that local `data` array?
Also, `Common:swap<sizeof(T)>` or `Common::BigEndianValue` or similar could be used instead of that loop.